modules/database/src/database.proto

```diff
@@ -62,6 +62,26 @@ message QueryRequest {
     string query = 2;
 }
 
+message CombinedRawQueryRequest {
+    string schemaName = 1;
+    rawQuery query = 2;
+}
+
+message rawQuery {
```

Comment:
Protobuff message names should be PascalCase.
`RawMongoQuery` and `RawSqlQuery` definitions can be nested inside `RawQuery`.